Karamelfile

```diff
@@ -2,6 +2,7 @@ dependencies:
   - recipe: flink::install
     global:  
       - kagent::install
+      - hops::install
```

Comment:
Karamel 0.6 does not respect install dependencies.

Reply:
Sure, but it's nice to had it there anyway in case some day Karamel supports that